### DISCUSSION TITLE
repeat use of CURL handle

### DISCUSSION THREAD

**Discussion by shunli0726** [curl/curl#6384] [Q&A] [answered]
auto curl = curl_easy_init()

can I repeat use the curl var when call   the function curl_easy_reset()   after 

for example :

auto curl = curl_easy_init()
 curl_easy_reset(curl )

next frame to continue use the curl

**Comment by bagder** [comment, marked as answer]
Yes, you can use the `CURL *` handle again in repeated transfers. In fact, you're encouraged to!
